library/src/main/java/com/mapzen/valhalla/Router.kt

```diff
@@ -56,6 +56,7 @@ interface Router {
             city: String? = null,
             state: String? = null): Router
     fun setDistanceUnits(units: DistanceUnits): Router
+    fun setMaxDifficulty(difficulty: Int): Router
```

Comment:
Could we update this to be called `setMaxHikingDifficulty` so its clear it affects hiking trails only and is consistent with the JSON